### ISSUE TITLE
pandas version

### ISSUE THREAD

**Issue by Yuee98** [pan324/anno1800_seedfinder#9]
I got an error showing:
```
  File "E:\anno1800_seedfinder\main.py", line 205, in <module>
    oldworld, cape, newworlds, oldislands, newislands = Load(maptype, mapsize, islandsize, difficulty, gamemode, dlc12)
  File "E:\anno1800_seedfinder\util.py", line 63, in Load
    maps = pickle.load(open("maps/pickle","rb"))
AttributeError: Can't get attribute '_unpickle_block' on <module 'pandas._libs.internals' from ...
``` 
Seems it's a pandas version problem. 
Could you provide the version of pandas that created the pickle file?

**Comment by Yuee98** [comment]
fixed with pandas==1.4.1

**Comment by pan324** [comment]
Ooh, I had totally missed this. I guess I will take another look in order to use simpler data instead of pandas.
